v1/atm/atm.gunk

```diff
@@ -89,55 +87,60 @@ type CreateATMRequest struct {
 	// BankID is the bank identifier that owned the ATM
 	BankID string `pb:"1" json:"bank_id"`
 
-	// Name is the name of ATM
+	// BankID is the identifier of the bank associated with the ATM.
 	Name string `pb:"2" json:"name"`
 
-	// Address is the ATM location details
+	// Address is the ATM's address.
 	Address types.Address `pb:"3" json:"3"`
 
-	// Location is the ATM longitude and latitude
+	// Location is the ATM's longitude and latitude
 	Location types.Location `pb:"4" json:"location"`
 
-	// Description is the ATM description
+	// Description is the ATM's description.
 	Description string `pb:"5" json:"description"`
 
-	// Metadata is the ATM metadata
+	// Metadata is the ATM's metadata.
 	Metadata string `pb:"6" json:"metadata"`
 }
 
-// CreateATMResponse is a response from CreateATMRequest
+// CreateATMResponse is the response envelope for creating an ATM.
 type CreateATMResponse struct {
-	// ATM_ID is a unique identifier of the ATM.
+	// ATM_ID is the unique identifier of the ATM.
 	ATM_ID string `pb:"1" json:"atm_id"`
 }
 
-// GetATMRequest is a request to get the ATM data.
+// GetATMRequest is the request envelope for retrieving a specific ATM's information.
 type GetATMRequest struct {
 	// ATM_ID is a unique identifier of the ATM.
 	ATM_ID string `pb:"1" json:"atm_id"`
 }
 
-// GetATMsResponse is the response for GetATMs
-type GetATMsResponse struct{
+// GetATMsResponse is the response envelope for retrieving ATM information.
+type GetATMsResponse struct {
 	// Result is the list of the ATMs.
 	Result []ATM `pb:"1" json:"result"`
 }
 
 // ATMService provides ATM operations.
 type ATMService interface {
-	// GetATM retrieves the detail of an ATM information, selected by its ID.
+	// GetATM retrieves the details regarding an specific ATM.
 	//
 	// +gunk http.Match{
 	//         Method: "GET",
 	//         Path:   "/v1/atms/{ATM_ID}",
 	// }
 	// +gunk openapiv2.Operation{
 	//         Tags:        []string{"ATM"},
-	//         Description: "Retrieves ATM information, selected by the ATM_ID you supplied.",
-	//         Summary:     "Retrieve ATM information",
+	//         Description: "Retrieves information regarding a specific ATM, selected by the supplied ID.",
+	//         Summary:     "Retrieve an ATM",
 	//         Responses: map[string]openapiv2.Response{
 	//                 "200": openapiv2.Response{
 	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/atmATM",
```

Comment:
Should the Ref be "#/definitions/atm`s`ATM"?

Reply:
no, since the package name is singular: https://github.com/openbank/openbank/blob/b496bf680a51b141c09f9cb480b133d4d54aba6e/v1/atm/all.swagger.json#L173

Reply:
the gunk http path on each function starts with `/atms/...`? if it's meant to be singular, should that be `/atm/...`?

Reply:
i understand your concern, i do think the `atm` package might need to be renamed for consistency with the other packages (which are plural), but any non-documentation changes are out of scope for this PR. it can be addressed in a future PR